tests/integration/study_data_blueprint/test_table_mode.py

```diff
@@ -188,25 +189,176 @@ def test_lifecycle__nominal(
         assert res.status_code == 200, res.json()
         actual = res.json()
         assert set(actual["properties"]) == {
-            "colorRgb",
-            "comments",
-            "hurdlesCost",
-            "loopFlow",
-            "usePhaseShifter",
-            "transmissionCapacities",
             "assetType",
-            "linkStyle",
-            "linkWidth",
+            "colorb",
+            "colorg",
+            "colorr",
             "displayComments",
             "filterSynthesis",
             "filterYearByYear",
+            "hurdlesCost",
+            "linkStyle",
+            "linkWidth",
+            "loopFlow",
+            "transmissionCapacities",
+            "usePhaseShifter",
         }
 
-        res = client.put(
-            f"/v1/studies/{internal_study_id}/table-mode/links",
-            json={
+        if study_version < STUDY_VERSION_8_2:
```

Comment:
Why do you test it only after v8.2 ? Is it because of the filterSynthesis/yearByYear attributes  ? If so, you could just adapt the expected response according to the version but perform this test for all versions